test/web-platform-tests/to-run.yaml

```diff
@@ -521,7 +521,6 @@ date.html: [fail, Unknown]
 datetime-local.html: [fail, Unknown]
 datetime.html: [fail, Unknown]
 email.html: [fail, Unknown]
-files.html: [fail, Unknown]
```

Comment:
Why are we removing this file?

Reply:
`to-run.yaml` lists all tests that are known to be failing. When I added the setter for `HTMLInputElement.files` this test began to pass and it could be removed from the list.